client/src/pages/playlists/playlistsPage.js

```diff
@@ -120,6 +120,7 @@ const PlaylistPage = () => {
             source={playlist.source}
             type={playlist.type}
             description={playlist.description}
+            playlistObject={playlist}
```

Comment:
not making more changes here since I'm assuming the inputs can clash with some other PRs. Should remvoe the params above later